entities/Organization.entity.json

```diff
@@ -22,14 +22,6 @@
         "text": {
           "$ref": "#/definitions/Narrative"
         },
-        "identifier": {
```

Comment:
The Org identifier is removed without a code-specific identifier added. That may be ok if the data set has no full IDs for orgs.